.github/workflows/test.yml

```
@@ -0,0 +1,94 @@
+name: Tests
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches-ignore:
+      - master
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+env:
+  DEFAULT_PYTHON: '3.11'
+
+jobs:
+  tests:
+    name: Run tests (Python 3.11, Pydantic v1 on ubuntu-latest)
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    env:
+      POETRY_VERSION: 1.7.1
+
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Build Worker image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          target: test
+          tags: syncmaster_worker:${{ github.sha }}
+          file: docker/worker.dockerfile
+          cache-to: type=gha,mode=max
```

```suggestion
          pull: true
          cache-to: type=gha,mode=max
```
Может на разных воркерах образы Python разных patch версий, и сборка из-за этого падает с какой-то дичью в логах